spec.md

```diff
@@ -0,0 +1,954 @@
+# Container Machine Interface (CMI)
```

Comment:
Any suggestions for a better name are welcome here. Preferably something along the lines of CMI.

Reply:
How about CMI but *Common* Machine Interface? 😄

Reply:
`Container Machine Interface` is perfect IMO.